$NetBSD: patch-aa,v 1.3 2011/01/19 13:18:27 adam Exp $

Fix problem with gettext definition of LC_* variables. See 
code below as example of warning. 

perl -MLocale::gettext -MPOSIX -e ''
Prototype mismatch: sub main::LC_ALL: none vs () at -e line 0
Prototype mismatch: sub main::LC_NUMERIC: none vs () at -e line 0
Prototype mismatch: sub main::LC_TIME: none vs () at -e line 0
Prototype mismatch: sub main::LC_MONETARY: none vs () at -e line 0
Prototype mismatch: sub main::LC_CTYPE: none vs () at -e line 0
Prototype mismatch: sub main::LC_MESSAGES: none vs () at -e line 0
Prototype mismatch: sub main::LC_COLLATE: none vs () at -e line 0

Problem fixed with patch from 

http://rt.cpan.org/Public/Bug/Display.html?id=35680

--- gettext.pm.orig	2005-06-01 03:11:16.000000000 +0000
+++ gettext.pm
@@ -32,6 +32,7 @@ to internationalize software.
 =cut
 
 use Carp;
+use POSIX qw(:locale_h);
 
 require Exporter;
 require DynaLoader;
